.github/workflows/translations.yml

```diff
@@ -0,0 +1,62 @@
+name: Make Translation
+
+on: ["push", "pull_request"]
+# on:
+#  push:
+#    branches:
+#      - master
+
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      INVENTREE_DB_NAME: './test_db.sqlite'
+      INVENTREE_DB_ENGINE: django.db.backends.sqlite3
+      INVENTREE_DEBUG: info
+      INVENTREE_MEDIA_ROOT: ./media
+      INVENTREE_STATIC_ROOT: ./static
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: get current translations
+      run: |
+        git fetch
+        git checkout origin/l10_base -- `git ls-tree origin/l10_base -r --name-only | grep ".po"`
+        git reset
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install Dependencies
+      run: |
+          sudo apt-get update
```

Comment:
As the translation (`*.po`) files are already committed, do we need to run `invoke translate` as a CI step?

Reply:
- it first overrides the *.po-files to keep the changes in the structure of the po files as small as possible
- make translations to adapt the po
- push the updated po-file